examples/js/postprocessing/AdaptiveToneMappingPass.js

```diff
@@ -186,7 +186,7 @@ THREE.AdaptiveToneMappingPass.prototype = {
 			this.previousLuminanceRT.dispose();
 
 		}
-		var pars = { minFilter: THREE.LinearFilter, magFilter: THREE.LinearFilter, format: THREE.RGBFormat };
+		var pars = { minFilter: THREE.LinearFilter, magFilter: THREE.LinearFilter, format: THREE.RGBAFormat };
```

Comment:
[this](https://bugzilla.mozilla.org/show_bug.cgi?id=1233046) bug was fixed today, and its duplicates 1245631 mention code 0x8cd6 and 1245850 mention GL_RGBA :) that's some coarse match, could not be bothered to read details